.devcontainer/docker-compose-windows.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.9"
```

Comment:
I think version line is obsolete
https://dev.to/ajeetraina/do-we-still-use-version-in-compose-3inp